code/game/objects/items/storage/belt.dm

```diff
@@ -326,6 +326,52 @@
 	GET_COMPONENT(STR, /datum/component/storage)
 	STR.max_w_class = WEIGHT_CLASS_SMALL
 
+/obj/item/storage/belt/military/snack
+	name = "tactical snack rig"
+	var/list/stop_eating_like_a_noob = list(

```

Comment:
If this is only referenced by Initialize, there doesn't need to be a new instance for each belt. It can be made a static list.